test/common/http/header_map_impl_test.cc

```diff
@@ -469,14 +477,6 @@ TEST_P(HeaderMapImplTest, InlineAppend) {
     headers.setVia("2.0 override");
     EXPECT_EQ(headers.getViaValue(), "2.0 override");
   }
-  {
-    // Set and then append. This mimics how GrpcTimeout is set.
-    TestRequestHeaderMapImpl headers;
-    headers.setGrpcTimeout(42);
-    EXPECT_EQ(headers.getGrpcTimeoutValue(), "42");
-    headers.appendGrpcTimeout("s", "");
-    EXPECT_EQ(headers.getGrpcTimeoutValue(), "42s");
-  }
```

Comment:
Why remove this? Is this covered by another test?

Reply:
Yep, it's behavior is covered by `Via`: https://github.com/envoyproxy/envoy/blob/d0ae54158773ea9ac971286de606ee2b54c784d3/test/common/http/header_map_impl_test.cc#L447

It was meant to test how grpc-timeout is specifically set here https://github.com/envoyproxy/envoy/blob/1d1b708c7bf6efa02c41d9ce22cbf1e4a1aeec2c/source/common/grpc/common.cc#L223, but that's not how grpc-timeout is set anymore.

Reply:
Thanks for explaining!